CHANGELOG.rst

```diff
@@ -6,6 +6,8 @@ RTOS Framework change log
 
   * FIXED: Added dummy definitions of vTaskCoreAffinitySet() and vTaskCoreAffinitySet() functions to
     avoid warnings when configNUM_CORES is set to 1.
+  * UPDATED: To fwk_io v3.3.0 from v3.0.1 for CI run
+  * UPDATED: To fwk_core v1.0.2 from v1.0.0 for CI run
```

Comment:
Do we list changes related to the tests in the changelog?

Reply:
I don't see why not - I think it's useful info for the user to understand what it is verified against